### DISCUSSION TITLE
How to save results in .srt(timestamps) file in python programmatically

### DISCUSSION THREAD

**Discussion by eliich** [openai/whisper#2029] [Q&A]
Hello all. Basically I am trying to make this python script instead of console.logging I would like it to save the segmented results into a .srt file. But currently when I console log it its just in pure text, non segmented format
```
import whisper

# Specify the full path of your audio file using raw string notation
audio_file_path = r"C:\Users\USER\Desktop\MikeQuinnPostVoiceRemoval.wav"

# Load the model
model = whisper.load_model("base")

# Transcribe the audio
result = model.transcribe(audio_file_path)

# Print the transcribed text
print(result["text"])
```
Which I call using this python interpreter
```
& C:/Users/USER/anaconda3/envs/whisper/python.exe c:/Users/USER/Programming/AI/Speech-to-Text/Transcription/transcribe.py
```

**Comment by glangford** [comment]
```python
# write srt to the current directory
# writer will use the same base name as the audio file with '.srt' suffix
writer = get_writer("srt", ".") 
writer(result, audio_file_path, {"max_line_width":55, "max_line_count":2, "highlight_words":False} )
```

**Reply by lukeewin** [reply]
Thank you very much, but the default is an underscore. If I want to customize it to another style, how should I resolve this?

**Reply by glangford** [reply]
I don't understand your question. What default are you referring to?

**Reply by acidburnmonkey** [reply]
from what module is  this get_writer() coming from ?

**Reply by glangford** [reply]
`get_writer()` is in utils.py

https://github.com/openai/whisper/blob/517a43ecd132a2089d85f4ebc044728a71d49f6e/whisper/utils.py#L296
